### DISCUSSION TITLE
Checking ARC version and upgrading

### DISCUSSION THREAD

**Discussion by saifur-m** [actions/actions-runner-controller#1808] [Questions] [answered]
Hello,

I would like to know what is the difference between [v0.26.0](https://github.com/actions-runner-controller/actions-runner-controller/releases/tag/v0.26.0) and [actions-runner-controller-0.21.0](https://github.com/actions-runner-controller/actions-runner-controller/releases/tag/actions-runner-controller-0.21.0).

How do I check the current version of ARC and what is the best way to upgrade?

**Comment by toast-gear** [comment, marked as answer]
There are multiple release streams in the repository. The former is the controller + CRDs itself, the latter is the Helm chart
